src/features/campaigns/components/CampaignDetailsForm.tsx

```diff
@@ -93,39 +84,6 @@ const CampaignDetailsForm = ({
             variant="outlined"
           />
 
-          {/* This is broken due to MUIv5 migration, but will be replaced by new UI
-          <ZUIPersonSelect
-            label={intl.formatMessage({
-              id: 'misc.formDialog.campaign.manager.label',
-            })}
-            name="manager_id"
-            onChange={(person) => {
-              setSelectedManager(person);
-            }}
-            selectedPerson={selectedManager}
-          />
-          */}
-
-          {userProfile && userProfile.id != selectedManager?.id && (
-            <Link
-              color="textPrimary"
-              component="button"
-              onClick={() => {
-                // Select profile beloning to current user. We only need the ID
-                // to be correct, and the name to be reflected by first_name+last_name,
-                // so for simplicity we pass the entire name as first name.
-                setSelectedManager({
-                  first_name: userProfile.name,
-                  id: userProfile.id,
-                  last_name: '',
-                } as ZetkinPerson);
-              }}
-              underline="hover"
-            >
-              <Msg id={messageIds.form.manager.selectSelf} />
```

Comment:
Maybe you can find and remove the messageId as well? Hint: Use your IDE to find the definition of `selectSelf`, or go look in the `messageIds.ts` module of this feature (`campaigns`).

Reply:
Removed it from messageIds.ts.

I did not remove the strings for the different locale files because those are generated, right?